app/src/main/res/values/strings.xml

```diff
@@ -162,4 +161,7 @@
     <string name="guide">Návod</string>
     <string name="unexpected_error_text">Nastala neočekávaná chyba. Zkuste to později. Pokud se chyba opakuje delší dobu, konktaktujte naší podporu.</string>
     <string name="data_description">O zobrazených datech</string>
+    <string name="verify_later_desc">Nedorazila vám SMS zpráva s ověřovacím kódem? Začněte aplikaci používat hned a ověřte telefonní číslo později.</string>
+    <string name="verify_later_button">Dokončím ověření později</string>
+    <string name="verify_phone_number">ověřit telefonní číslo</string>
```

Comment:
Nemělo by to být _Ověřit telefonní číslo_ (capital O) ?

Reply:
Button dělá all caps, takže cajk